scripts/configure-kueue.sh

```diff
@@ -0,0 +1,113 @@
+#!/bin/bash
+
+# Kueue Configuration Script
+# Configures a single Kueue cluster with ClusterQueue, LocalQueue, and ResourceFlavor
+
+set -e
+
+# Colors for output
+GREEN='\033[0;32m'
+YELLOW='\033[1;33m'
+BLUE='\033[0;34m'
+NC='\033[0m' # No Color
+
+echo -e "${BLUE}⚙️  Configuring Kueue environment${NC}"
+echo "================================="
+
+# Function to print status
+print_status() {
+    echo -e "${GREEN}✅ $1${NC}"
+}
+
+run_cmd() {
+    echo -e "${YELLOW}$ $*${NC}"
+    "$@"
+}
+
+# Step 1: Install Kueue
+echo -e "${BLUE}📦 Installing Kueue...${NC}"
+run_cmd helm install kueue oci://registry.k8s.io/kueue/charts/kueue \
+    --version=0.13.2 \
+    --namespace kueue-system \
+    --create-namespace \
+    --wait --timeout 300s
+print_status "Kueue installed"
+
+# Step 2: Apply manifests directly from inline YAML
+echo -e "${BLUE}📄 Applying Kueue resource manifests...${NC}"
+kubectl apply -f - <<'EOF'
+---
+apiVersion: kueue.x-k8s.io/v1beta1
+kind: ResourceFlavor
+metadata:
+  name: "default-flavor"
+---
+# Local queue for batch jobs
+apiVersion: kueue.x-k8s.io/v1beta1
+kind: LocalQueue
+metadata:
+  namespace: "default"
+  name: "local-queue-batch"
+spec:
+  clusterQueue: "cluster-queue-reana-run-batch"
+---
+# Cluster queue for batch jobs
+apiVersion: kueue.x-k8s.io/v1beta1
+kind: ClusterQueue
+metadata:
+  name: "cluster-queue-reana-run-batch"
+spec:
+  namespaceSelector: {} # match all.
+  resourceGroups:
+  - coveredResources: ["cpu", "memory"]
+    flavors:
+    - name: "default-flavor"
+      resources:
+      - name: "cpu"
+        nominalQuota: 3
+      - name: "memory"
+        nominalQuota: 4Gi
```

Comment:
For run-batch, we can have less CPU and less RAM value configured. This will provide a better example why we need two queues. (Small pods reana-run-batch running just workflow orchestration, large pods reana-run-job running lengthy user payload.) You can try to decease reana-run-batch queue resources and increase reana-run-job queue resources.